src/Meilisearch/MeilisearchClient.cs

```diff
@@ -122,34 +122,27 @@ public async Task<TaskInfo> DeleteIndexAsync(string uid, CancellationToken cance
         /// </summary>
         /// <param name="cancellationToken">The cancellation token for this call.</param>
         /// <returns>An IEnumerable of indexes in JsonElement format.</returns>
-        public async Task<IEnumerable<JsonElement>> GetAllRawIndexesAsync(CancellationToken cancellationToken = default)
+        public async Task<JsonDocument> GetAllRawIndexesAsync(CancellationToken cancellationToken = default)
         {
             var response = await _http.GetAsync("indexes", cancellationToken).ConfigureAwait(false);
 
             var content = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
-            var json = JsonDocument.Parse(content);
-            var indexes = new List<JsonElement>();
-
-            foreach (var element in json.RootElement.EnumerateArray())
-            {
-                indexes.Add(element);
-            }
-
-            return indexes;
+            return JsonDocument.Parse(content);
```

Comment:
I love the simplification, but I would love more when we drop these raw methods 😬

Reply:
I know I hesitate to remove it too ...